views/templates/front/paymentOptions/paypal.tpl

```diff
@@ -58,7 +58,7 @@ function initPaypalScript() {
   }
 
   const paypalScript = document.createElement('script');
-  paypalScript.setAttribute('src', "https://www.paypal.com/sdk/js?components=hosted-fields,buttons&client-id={$paypalClientId|escape:'htmlall':'UTF-8'}&merchant-id={$merchantId|escape:'htmlall':'UTF-8'}&intent={$intent|escape:'htmlall':'UTF-8'}&currency={$currencyIsoCode|escape:'htmlall':'UTF-8'}&locale={$locale|escape:'htmlall':'UTF-8'}");
+  paypalScript.setAttribute('src', "{$paypalSdkLink nofilter}");
```

Comment:
Why did you use nofilter here ?

Reply:
Because without url is encoded like `https://www.paypal.com/sdk/js?components=buttons&amp;locale=fr_FR&amp;currency=EUR&amp;intent=capture&amp;disable-funding=card` and a javascript error 👍